fitbit-react/src/Util/Toasts/Toasts.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from "react";
+import { useToast } from "../../Context/toast-context";
+import { AiFillCloseSquare } from "../../Assets/icons";
+import "./toasts.css";
+
+function Toasts() {
+  const { toastList, dispatchToast } = useToast();
+
+  const deleteToast = (id) => {
+    let newToastList = toastList.filter((toast) => toast.id !== id);
+
+    dispatchToast({
+      type: "SET_LIST",
+      payload: newToastList,
```

Comment:
suggestion: better naming like 'SET_TOAST_TO_THE_LIST' helps to understand easily.

Reply:
I feel this name would be very big that is why i kept like that